salesforce-gui/src/test/java/org/fundacionjala/salesforce/cucumber/stepdefs/AccountSteps.java

```diff
@@ -0,0 +1,117 @@
+package org.fundacionjala.salesforce.cucumber.stepdefs;
+
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import io.restassured.response.Response;
+import org.fundacionjala.core.api.client.RequestManager;
+import org.fundacionjala.salesforce.utils.ApiResponseDataExtractor;
+import org.fundacionjala.salesforce.constants.AccountConstants;
+import org.fundacionjala.salesforce.ui.context.Context;
+import org.fundacionjala.salesforce.ui.entities.Account;
+import org.fundacionjala.salesforce.ui.skins.ISkinFactory;
+import org.fundacionjala.salesforce.ui.skins.SkinManager;
+import org.fundacionjala.salesforce.utils.PageTransporter;
+import org.testng.asserts.SoftAssert;
+
+import java.net.MalformedURLException;
+import java.util.Map;
+
+/**
+ * [MR] StepDefinitions class for salesforce Accounts.
+ */
+public class AccountSteps {
+
+    private Account account;
+
+    private ISkinFactory skin = SkinManager.getInstance().getSkinFactory();
+
+    //Context
+    private final Context context;
+
+    /**
+     * Adds Dependency injection to share Context information.
+     * @param sharedContext
+     */
+    public AccountSteps(final Context sharedContext) {
+        this.context = sharedContext;
+    }
+
+    /**
+     * Creates an account from UI and update related account entity.
+     *
+     * @param accountInfo to create a new Account
+     */
+    @When("I create an Account with the following data")
+    public void createAnAccountWithTheFollowingData(final Map accountInfo) {
+        //Updating Entity
+        account = new Account();
+        account.setInformation(accountInfo);
+        account.setUpdatedFields(accountInfo.keySet());
+
+        skin.getAccountsPage().goToAccountCreation();
+        skin.getAccountCreationPage().fillAccountInformation(account.getUpdatedFields(), account);
+
+        account.setId(skin.getAccountDetailsPage().getAccountId());
+        context.setAccount(account);
+    }
+
+    /**
+     * Makes assertions for Account Details and the recently account entity.
+     */
+    @Then("Account's new data should be displayed at details")
+    public void accountSNewDataShouldBeDisplayedAtDetails() {
+        Map<String, String> actualAccountDetails = skin.getAccountDetailsPage().
+                getAccountDetails(account.getUpdatedFields());
+        Map<String, String> expectedAccountDetails = account.getAccountInfo();
+        SoftAssert softAssert = new SoftAssert();
+        actualAccountDetails.forEach((field, actualValue) -> {
+            softAssert.assertEquals(actualValue, expectedAccountDetails.get(field),
+            "The " + field + " from Account Details Page does not match with the " + field + " edited previously.");
```

Comment:
Another possiblñe solution is 

String.format("The %s from Account Details Page does not match with the %s edited previously.", field)